tests/handlers/contract-request.test.ts

```diff
@@ -627,8 +649,367 @@ describe('contract-request', () => {
     );
 
     expect(ciResponse).not.be.undefined;
-    expect((ciResponse.values().next().value as ZeroKnowledgeProofResponse).id).to.be.equal(
-      proofReqs[0].id
+    expect(
+      (
+        (ciResponse as Map<string, ZeroKnowledgeProofResponse>).values().next()
+          .value as ZeroKnowledgeProofResponse
+      ).id
+    ).to.be.equal(proofReqs[0].id);
+  });
+
+  // cross chain integration test
+  it.skip('cross chain contract request flow - integration test', async () => {
+    const privadoTestRpcUrl = '<>'; // issuer RPC URL - privato test
+    const privadoTestStateContract = '0x975556428F077dB5877Ea2474D783D6C69233742';
+    const amoyVerifierRpcUrl = '<>'; // verifier RPC URL - amoy
+    const erc20Verifier = '0x74030e4c5d53ef381A889C01f0bBd3B8336F4a4a';
+
+    const issuerStateEthConfig = defaultEthConnectionConfig;
+    issuerStateEthConfig.url = privadoTestRpcUrl;
+    issuerStateEthConfig.contractAddress = privadoTestStateContract; // privado test state contract
+
+    const memoryKeyStore = new InMemoryPrivateKeyStore();
```

Comment:
we have helper function registerKeyProvidersInMemoryKMS and getInMemoryDataStorage in helper.ts. It will help to make this test code size shorter

Reply:
reused